contracts/token/ERC721/extensions/ERC721URIStorage.sol

```diff
@@ -15,7 +15,7 @@ abstract contract ERC721URIStorage is IERC4906, ERC721 {
     using Strings for uint256;
 
     // Optional mapping for token URIs
-    mapping(uint256 => string) private _tokenURIs;
+    mapping(uint256 tokenId => string uri) private _tokenURIs;
```

Comment:
Same here, that is an extension. 

I'm also nothing that the ERC1155 implementation is cleaner, maybe we want to reproduce it here.

Reply:
Same as https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4433#discussion_r1256692650